src/Altinn.Platform/Altinn.Platform.Storage/Storage/appsettings.Staging.json

```diff
@@ -0,0 +1,9 @@
+{
+  "Logging": {
```

Comment:
Jeg er litt usikker på om dette faktisk gir noe effekt git oppsettet på Application Insight Logging

Reply:
Yeah, the builder.ClearProviders() call probably makes these settings redundant, but I wasn't 100% certain so I decided to copy settings from dev.